tools/pull_requests/sme_groups.json

```diff
@@ -0,0 +1,43 @@
+{
+    "APIs": [],
+    "Backup": [],
+    "Block cache": [],
+    "Block manager": ["etienneptl"],
+    "Btree": [],
+    "Build-compile": [],
+    "Cache and eviction": ["ajmorton"],
+    "Checkpoints": [],
+    "Column store": [],
+    "Command line utilities": [],
+    "Compaction": [],
+    "Config Parsing": [],
+    "cppsuite": [],
+    "csuite": [],
+    "Cursor implementations": [],
+    "dhandles": [],
+    "Evergreen": [],
+    "Example code": [],
+    "Extensions": [],
+    "Filesystem API": [],
+    "gdb-scripts": [],
+    "Language bindings": [],
+    "Lint": [],
+    "Logging": [],
+    "LSM": [],
+    "none": [],
+    "Overflow Items": [],
+    "Platform": [],
+    "prepared-txns": [],
+    "Python tests": [],
+    "Reconciliation": [],
+    "RTS": [],
+    "Salvage": [],
+    "Schema management": [],
+    "Test Format": [],
+    "testy": [],
+    "Truncate": [],
+    "Verify": [],
+    "WT Docs": [],
+    "wt-notfound": [],
```

Comment:
What is this one?

Reply:
I pulled this list directly from the Jira ticket component list. From memory via a REST api call. I'll remove it since it doesn't seem relevant.